stdlib/list.ml

```diff
@@ -228,6 +228,14 @@ let rec find_opt p = function
   | [] -> None
   | x :: l -> if p x then Some x else find_opt p l
 
+let rec find_map f = function
+  | [] -> None
+  | x :: l ->
+     begin match f x with
```

Comment:
Style: `filter_map` below has no `begin` `end` wrapping in the same case (do as you wish just saying...)